src/Symfony/Component/Console/Application.php

```diff
@@ -1098,6 +1098,16 @@ public function setDefaultCommand($commandName, $isSingleCommand = false)
         return $this;
     }
 
+    /**
+     * Returns whether the application is a single command application or not.
+     *
+     * @return bool Whether the application is a single command application or not
+     */
+    public function isSingleCommandApplication()
```

Comment:
I suggest `isSingleCommand(): bool`. Adding the return type will remove the need for the `@return` phpdoc annotation